draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -4068,9 +4068,10 @@ Content-Range: exampleunit 11.2-14.3/25
    <x:ref>Range</x:ref> header field in the request (<xref target="header.range"/>).
 </t>
 <t>
-   A payload within a GET request message has no defined semantics;
-   sending a payload body on a GET request might cause some existing
-   implementations to reject the request.
+   A payload within a GET request message has no defined semantics. A client
+   &SHOULD-NOT; generate a body in a GET request. Sending a payload body on a
+   GET request might cause some existing implementations to reject the
+   request.
```

Comment:
"... or might cause surprising effects in HTTP caches which ignore the payload."

Reply:
perhaps "... or might cache caches to reuse the response erroneously."